### ISSUE TITLE
Villagers sleep with eyes open

### ISSUE THREAD

**Issue by RensSwag2001** [TangoTek/TekTopia-Community#245]
**Describe the bug**
The butcher sleeps with his/her eyes open

**Expected behavior**
Just like all other villagers, sleep with their eyes closed.

**Screenshots**
![2019-06-01_14 45 41](https://user-images.githubusercontent.com/51168774/58748606-b55b1580-847b-11e9-8eab-24e4e3b2f6ad.png)

** Server/Location **
Server 1
Town hall token: -1341 67 -8808

#208 looks like my post!

**Comment by JohnyDL** [comment]
Yup duplicate of #208 (you could have added it to the comments... I didn't see this villager when I did my check

**Comment by RensSwag2001** [comment]
@JohnyDL theyre going to sleep in a sec now.. maybe watch?
